docs/stencil-docs/page-builder/configuration.md

```diff
@@ -0,0 +1,155 @@
+* Merchants must select one variation to edit at a time. The selections that they make in the remainder of Page Builder's UI will apply only to that selected variation.
+* Page Builder's remaining sequence of top-level section headings corresponds directly to the top-level section objects that you declare in `schema.json`. The options displayed within these expandable section headings correspond directly to the key-value pairs that you nest within `schema.json`'s corresponding section objects.
+
+In all, the structure that you give your theme's `config.json` and `schema.json` files directly governs the UI that Page Builder exposes to merchants. 
+
+## Page Builder data types
+
+You are free to decide which properties of your theme to make editable in Page Builder and in which order to display them. Page Builder can expose any set of properties as long as your `schema.json` declares them using the data types that Page Builder supports.
+
+Page Builder supports the following data types:
+* color
+* font
+* select [drop-down list]
+* checkbox
+* imageDimension
+* text
+
+Within `schema.json`, each object's data type is declared in a statement like the one highlighted here:
+
+```json
+ {
+    "type": "color",
+    "label": "Text Color",
+    "id": "body-font-color"
+ },
+```
+
+### Types versus "heading" labels
+
+Within `schema.json`, you will also see `"type": "heading"` statements like the one below.
+
+```json
+{
+    "name": "Colors",
+    "settings": [
+      {
+        "type": "heading",
+        "content": "General"
+      },
+      {
+        "type": "color",
+        "label": "Text Color",
+        "id": "body-font-color"
+      },
+    {...}
+     ]
+}
+```
+
+These `"type": "heading"` statements do not reference data types. Rather, they declare display captions for the Page Builder UI's subcategories – the middle level nested within the section headings, but outside the individual options from which merchants can select. Inner options are designated by `"label": <label-text>` statements.
+
+## Page Builder data structure in `schema.json` 
+
+The `schema.json` nesting structure maps directly to the Page Builder UI displayed to merchants. Below the `variations` section, which contains data imported from `config.json`, the order and nesting of Page Builder's UI options directly match the order and nesting of your `schema.json` entries.
+
+## Best practices
+
+To prevent errors upon theme upload and avoid possible loss of customizations made through the Page Builder UI at runtime, please follow these guidelines:
+
+* **Single-instance restriction per storefront.** We strongly recommend opening only one instance of Page Builder at a time against each storefront. We give this recommendation because there is currently no synchronization mechanism to reconcile configuration changes made by multiple Page Builder instances. `schema.json` will record the last changes made by any instance, but changes saved earlier by other instances might be lost.
```

Comment:
```suggestion
* **Single-instance restriction per storefront.** We strongly recommend opening only one instance of Page Builder at a time against each storefront. We recommend this because there is currently no synchronization mechanism to reconcile configuration changes made by multiple Page Builder instances. The `schema.json` file will record the last changes made by any instance, but changes saved earlier by other instances might be lost.
```